src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
+import core.basesyntax.machines.Bulldozer;
+import core.basesyntax.machines.Excavator;
+import core.basesyntax.machines.Truck;
+import core.basesyntax.workflow.Machine;
+import core.basesyntax.workflow.MachineWorkingStatus;
+import java.util.List;
+
 public class MainApp {
+    public static void main(String[] arguments) {
+        List<Machine> machines = List.of(new Excavator(), new Bulldozer(), new Truck());
+        for (Machine machine : machines) {
+            MachineWorkingStatus.startWorkingDay(machine);
+        }
+        for (Machine machine : machines) {
+            MachineWorkingStatus.endWorkingDay(machine);
+        }
```

Comment:
let's remove class `MachineWorkingStatus` and call methods `doWork()` and `doStop()` directly